package-lock.json

```diff
@@ -7016,7 +7016,8 @@
             },
             "ansi-regex": {
               "version": "2.1.1",
-              "bundled": true
+              "bundled": true,
```

Comment:
I do not think there should be changes on this file. If it is needed, please specify why in a distinct commit with proper commit message.